files/image_config/rsyslog/rsyslog.d/00-sonic.conf.j2

```diff
@@ -10,26 +10,35 @@ template(name="prog_msg" type="list") {
 
 $ModLoad omprog
 
+{% endif %}
+{% endif %}
+
 if re_match($programname, "bgp[0-9]*#(frr|zebra|staticd|watchfrr)") then {
     /var/log/frr/zebra.log
+    {% if include_system_eventd == "y" %}
+    {% if build_reduce_image_size != "y" or sonic_asic_platform != "broadcom" %}
```

Comment:
Why have a ASIC check for Broadcom here?

Reply:
eventd should not be enabled on 7050qx and 7060cx3, so we check flags build_reduce_image_size for slim image enabled and also for broadcom asic.

Reply:
Why single flag is not enough? for example `build_reduce_image_size`?

Reply:
Initial requirement was for disabling eventd in 7050qx/7060 which is why I added a tighter check.